### PR TITLE
Devx 8159

### PR DESCRIPTION
**Pull request by gawaineo**
- added output of directives returned by Alexa
- refactored a few functions

### CODE REVIEW

File: alexa_client/alexa_client.py [hunk 1]
```diff
@@ -90,10 +90,11 @@ def get_request_params(self):
         }
         return url, headers, request_data
 
-    def save_response_audio(self, res, save_to=None):
+    def process_alexa_response(self, res, save_to=None):
```
Comment (bdibona-sonos) [inline alexa_client/alexa_client.py]: Will setting save_to=None override any value that is passed in this method?

File: alexa_client/alexa_client.py [hunk 2]
```diff
@@ -224,9 +235,9 @@ def ask_multiple(self, input_list):
             # Get the response from each future and save the audio
             for future, name_out in futures:
                 res = future.result()
-                save_to = self.save_response_audio(res, name_out)
-                saved_filenames.append(save_to)
-            return saved_filenames
+                response = self.process_alexa_response(res, name_out)
```
Comment (bdibona-sonos) [inline alexa_client/alexa_client.py]: Optional: Reconsider the variables res and response. At first glance they seem like the same thing.

Comment (gawaineo) [inline alexa_client/alexa_client.py]: Will look into this.

**Comment by carney-sonos** [comment]
Ship it!

**Pull request merged**